lib/chai-sorted.js

```diff
@@ -1,7 +1,9 @@
 'use strict'
 var isSorted = require('./is.sorted')
 
-var chaiIsSorted = function (chai, array, useDescendingOrder) {
+var chaiIsSorted = function (chai, array, options) {
+  var useDescendingOrder = typeof options !== 'undefined' ? !!options.descending : false
```

Comment:
What do you think about doing

```
var useDescendingOrder = typeof options === 'object' ? !!options.descending : false`
// Backwards compatibility
if (typeof options === 'boolean' ) {
  useDescendingOrder = options
}
```

Really I should expose the `chaiIsSorted` method for testing and can do so after this PR

Reply:
@jamwaffles I like this idea because it allows us to introduce `compareFunction` as a parameter which is similar to `Array.prototype.sort([optional compareFunction])`

Reply:
Hmm, that would ease the upgrade path for users yes. I'll add that in - thanks!